client/payment-details/summary/style.scss

```diff
@@ -98,4 +98,30 @@
 
 .payment-details__refund-controls {
 	flex: 0 0 auto;
+
+	.refund-controls__dropdown-menu {
+		/**
+		 * HACK: The following styles are needed to make the dropdown menu
+		 * appear in its expected position. The dropdown menu is positioned absolutely, so we need to make sure
+		 * that the parent container is positioned relatively.
+		 * This should be taken care of by the dropdown menu component's CSS, but we seem to be relying in outdated
+		 * wordpress/components styles.
+		 * TODO: link to GH issue.
```

Comment:
Will the link be added here?

Reply:
Yes, I'll add the link before merge  thanks!

Reply:
FYI: Added link to GH issue https://github.com/Automattic/woocommerce-payments/issues/8012. I'll fill it with more details as soon as possible